modules/trusted-firmware-m/Kconfig

```diff
@@ -64,6 +64,7 @@ config TFM_CRYPTO_BUILTIN_KEYS
 	prompt "TF-M crypto builtin keys"
 	default y
 	depends on !TFM_PROFILE_TYPE_MINIMAL
+	depends on !CRACEN_HW_PRESENT
```

Comment:
So we don't support TFM builtin keys with CRACEN, but we are modifying tfm_builtin_key_ids.h to support CRACEN?

What's the deal? :)